ext/openssl/ossl_bio.c

```diff
@@ -13,30 +13,34 @@ BIO *
 ossl_obj2bio(volatile VALUE *pobj)
 {
     VALUE obj = *pobj;
-    BIO *bio;
+    BIO *bio = NULL;
```

Comment:
As I commented on #442, I don't think we should do this.

Reply:
https://www.blackhat.com/presentations/bh-europe-06/bh-eu-06-Flake.pdf

Reply:
I would personally follow an established guideline for secure C programming. All the majors ones explicitly disallow uninitialised variables.

Reply:
Okay, this change is entirely reverted.